VulnerableApplication/Models/Hello.cs

```diff
@@ -11,5 +11,9 @@ public void Say()
         {
             Console.WriteLine("Hello!");
         }
+    public string Hi() {
+    return "this is against the rules.";
```

Comment:
[CODE_SMELL] 6 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.